README.md

```diff
@@ -99,6 +99,16 @@ which uses docker-compose for the web server as well.
 This will not cache dependencies - in particular, you'll have to rebuild all 400 whenever the lockfile changes -
 but makes sure that you're in a known environment so you should have fewer problems getting started.
 
+You can also use the `web` container to run builds on systems which don't support running builds directly (Mac OS, Windows and also some Linux setups):
```

Comment:
"some Linux setups" make it sound like almost nothing works, I would say "Linux without glibc" or something instead.

Reply:
I actually don't know which don't work :) Is it without `glibc`? 
I remembered @Nemo157 telling me that he has to run builds in the container too on his Linux setup.

Reply:
Nemo also configures his systems quite a lot I think 😆

Reply:
I removed the linux mention completely :)

Reply:
I think in my case it's my `~/.cargo/config` leaking into rustwide somehow, which indicates some issues with rustwide's sandboxing, I think it's actually ok now that I started using `CRATESFYI_RUSTWIDE_WORKSPACE=/tmp/docsrs-workspace` instead of having it in a sub-folder of `$HOME`.